documentation/common/snip-prerequisites-openshift.adoc

```diff
@@ -0,0 +1,14 @@
+:context: enmasse
+
+.Prerequisites
+
+* To install {ProductName}, the OpenShift client tools are required. You can download the OpenShift
+Origin client from link:https://github.com/openshift/origin/releases[OpenShift Origin^]. {ProductName} has
```

Comment:
This is only for the upstream.

Reply:
I'm not so sure about that, since {ProductName} is there. But it's probably not Origin that we want for downstream? @rgodfrey, do you know?